xml/System.Security.Cryptography.Pkcs/Rfc3161TimestampTokenInfo.xml

```diff
@@ -276,8 +296,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value indicating whether there are any extensions associated with this timestamp token.</summary>
+        <value>A value indicating whether there are any extensions associated with this timestamp token.</value>
```

Comment:
```suggestion
        <value><see langword="true"/> if there are any extensions associated with this timestamp token; <see langword="false"/> otherwise.</value>
```